src/test/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcherSpec.scala

```diff
@@ -271,6 +269,8 @@ class BlockFetcherSpec extends ScalaTestWithActorTestKit() with AnyFreeSpecLike
   }
 
   trait TestSetup extends TestSyncConfig {
+    val as: ActorSystem = ActorSystem("BlockFetcherSpec_System")
```

Comment:
Will it shutdown?

Reply:
eventually for sure, but please note that it's not any worse compared to the previous approach (the difference is the amount of systems, ~10 instead of 1) but the fundamental behaviour did not change. to build a proper "start/stop" loop would mean a refactor instead of a quick-fix for flakiness

Reply:
Well not arguing against this for a quick fix for the flakiness.

Hypothetically, in an interactive sbt session (eg. IntelliJ or Metals with SBT enabled, or just sbt) this could leak memory.